server/app.js

```diff
@@ -0,0 +1,75 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const mongoose = require('mongoose');
+const config = require('../config');
+require('../dataseeding/Schema/Review');
+require('../dataseeding/Schema/User');
+const Hostel = require('../dataseeding/Schema/Hostel');
+
+const PORT = process.env.PORT || 3004;
+
+const app = express();
+
+mongoose
+  .connect(config.MONGO_URI)
+  .then(() => console.log('Connected to database'))
+  .catch(err => console.log('Failed to connect to database:', err));
+
+// Body parser middleware
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
+
+app.get('/api/reviews/full/:hostelId', (req, res) => {
+  // console.log('hostelId', req.params.hostelId);
+  const { hostelId } = req.params;
```

Comment:
Does this work? Should it be req.params.hostelId? (same comment for second get request)

Reply:
@smrutherford yea it should work, I normally do as what you suggested but eslint made us deconstruct a key out of an object if you want a variable name same as the key.